indra/newview/llviewertexture.cpp

```diff
@@ -99,6 +99,7 @@ U32 LLViewerTexture::sMinLargeImageSize = 65536; //256 * 256.
 U32 LLViewerTexture::sMaxSmallImageSize = MAX_CACHED_RAW_IMAGE_AREA;
 bool LLViewerTexture::sFreezeImageUpdates = false;
 F32 LLViewerTexture::sCurrentTime = 0.0f;
+F32 LLViewerTexture::sFreeVRAMMegobytes = 768.0f;
```

Comment:
This is a pretty specific number and is worth documenting why it's used.

Reply:
sFreeVRAMMegabytes instead of sFreeVRAMMegobytes

Reply:
Declaread 768 as a const MIN_VRAM_BUDGET (value was selected due it being the minimum from updateClass())
Fixed name